File: app/src/main/java/kittentrate/data/mapping/PhotoEntityMapperInterface.java
package kittentrate.data.mapping;

import java.util.Collection;
import java.util.List;

import kittentrate.data.repository.model.Photo;
import kittentrate.data.repository.model.PhotoEntity;

/**
 * Created by Manuel Lorenzo on 20/03/2017.
 */

public interface PhotoEntityMapperInterface {
    List<PhotoEntity> transform(Collection<Photo> photoCollection);
}

File: app/src/main/java/kittentrate/data/mapping/PhotoEntityMapperInterface.kt
package kittentrate.data.mapping

import kittentrate.data.model.Photo
import kittentrate.data.model.PhotoEntity

/**
 * Created by Manuel Lorenzo on 20/03/2017.
 */

interface PhotoEntityMapperInterface {
    fun mapToEntity(photoCollection: List<Photo>): List<PhotoEntity>
}
